Drop column (#14824)
crate/crate: docs/sql/statements/alter-table.rst

      { SET ( parameter = value [ , ... ] )
        | RESET ( parameter [ , ... ] )
        | { ADD [ COLUMN ] column_name data_type [ column_constraint [ ... ] ] } [, ... ]

>         | { DROP [ COLUMN ] column_name } [, ... ]

Author: Added in https://github.com/crate/crate/pull/14824/commits/12d5d5e8d4fc98922166d927099c9b08a164976c

  Data type of the column which should be added.

:column_name:
  Name of the column which should be added.
  This can be a sub-column on an existing `OBJECT`.

It's possible to add multiple columns at once.

.. _sql-alter-table-drop-column:

``DROP COLUMN``
---------------

Can be used to drop a column from a table.

:column_name:

>   Name of the column which should be added.

Reviewer: ```suggestion
  Name of the column which should be dropped.
```

Author: Good catch!

>   This can be a sub-column on an existing `OBJECT`.

Reviewer: ```suggestion
  This can be a sub-column of an `OBJECT`.
```
